src/pages/Profile/Body/GeneralInfo/DetailsColumn/Details.tsx

```diff
@@ -20,7 +20,7 @@ export default function Details() {
         <Detail title="address">{profile.street_address}</Detail>
       )}
       <Detail title="active in">
-        {isEmpty(profile.active_in_countries)
+        {!!profile.active_in_countries && isEmpty(profile.active_in_countries)
```

Comment:
```suggestion
        {!profile.active_in_countries || isEmpty(profile.active_in_countries)
```

Reply:
If `active_in_countries` is undefined OR it is empty, show `HQ Country`.
If `active_in_countries` is defined and non-empty, show the list.